src/viur/core/modules/script.py

```diff
@@ -70,6 +71,15 @@ class ScriptLeafSkel(BaseScriptAbstractSkel):
         indexed=False
     )
 
+    access = SelectBone(
+        descr="Required access rights to run this Script",
+        values=lambda: {
+            right: translate("server.modules.user.accessright.%s" % right, defaultText=right)
```

Comment:
```suggestion
            right: translate(f"server.modules.user.accessright.{right}", defaultText=right)
```
@phorward  Why do you marked this as resolved?

Reply:
@ArneGudermann I've already changed this part of the code in 2dcde1530b1696e0e49c4b41e95dba3c9610c7e7 therefore I marked it as resolved.